src/istgtcontrol.c

```diff
@@ -158,25 +159,26 @@ typedef enum {
 	UCTL_CMD_CHAPSEQ = 6,
 } UCTL_CMD_STATUS;
 
-//#define ARGS_DELIM " \t\r\n"
-#define ARGS_DELIM " \t"
+//	#define ARGS_DELIM " \t\r\n"
+#define	ARGS_DELIM " \t"
 
 static int
 uctl_readline(UCTL_Ptr uctl)
 {
 	ssize_t total;
 
-	total = istgt_readline_socket(uctl->sock, uctl->recvbuf, uctl->recvbufsize,
+	total = istgt_readline_socket
```

Comment:
this doesn't look good. Just bring one parameter to next line, and next line should be given just 4 spaces extra, i.e., 4 spaces away under 'total'

Reply:
```
total = istgt_readline_socket (uctl->sock, uctl->recvbuf, uctl->recvbufsize,uctl->recvtmp, uctl->recvtmpsize,&uctl->recvtmpidx, &uctl->recvtmpcnt,
	    uctl->timeout);
```
Like this ?

Reply:
no.. we need it like this:
```
total = istgt_readline_socket (uctl->sock, uctl->recvbuf, uctl->recvbufsize,uctl->recvtmp,
    uctl->recvtmpsize,&uctl->recvtmpidx, &uctl->recvtmpcnt, uctl->timeout);
```